src/core/bau_inputs.jl

```diff
@@ -115,15 +121,18 @@ function BAUInputs(p::REoptInputs)
     t0, tf = p.s.electric_utility.outage_start_time_step, p.s.electric_utility.outage_end_time_step
     if tf > t0 && t0 > 0
         original_crit_lds = copy(p.s.electric_load.critical_loads_kw)
-        update_bau_outage_outputs(bau_scenario, original_crit_lds, t0, tf, production_factor)
+        generator_fuel_use_gal = update_bau_outage_outputs(bau_scenario, original_crit_lds, t0, tf, production_factor)
```

Comment:
Does the `generator_fuel_use_gal` now only apply for modeling single outages (not multiple with probabilities)?

Reply:
Right. The outage is only considered in the emissions calculations when a single outage is modeled. This is noted in the doc strings of the renewable energy calc/constraints functions. I also just added a note about it to the ElectricUtility doc string for documentation at a higher level.